src/plugins/data_views/server/rest_api_routes/runtime_fields/get_runtime_field.ts

```diff
@@ -43,17 +43,28 @@ export const getRuntimeField = async ({
   usageCollection?.incrementCounter({ counterName });
   const dataView = await dataViewsService.get(id);
 
-  const field = dataView.fields.getByName(name);
+  const field = dataView.getRuntimeField(name);
 
   if (!field) {
     throw new ErrorIndexPatternFieldNotFound(id, name);
   }
 
-  if (!field.runtimeField) {
-    throw new Error('Only runtime fields can be retrieved.');
+  // Access the data view fields created for the runtime field
+  let dataViewFields: DataViewField[];
```

Comment:
While the provided test case worked great, I tested with `params` in the payload and got this error:
```
{"statusCode":400,"error":"Bad Request","message":"[request body.runtimeField.script.params]: definition for this key is missing"}%                                                                                        
```

```
{
  "name": "composite_params",
  "runtimeField": {
    "type": "composite",
    "script": {
      "source": """emit("a","a"); emit("b","b");emit("c",params['c'])""",
      "params": {
        "c": "c"
      }
    },
    "fields": {
      "a": {
        "type": "keyword"
      },
      "b": {
        "type": "keyword"
      },
      "c": {
        "type": "keyword"
      }
    }
  }
}
```


Since this is a valid ES request, do we have any plans to support it in the future?

Reply:
>Since this is a valid ES request, do we have any plans to support it in the future?

I don't believe so, although I'm interested to hear arguments for it. Its unclear to me how this is useful, in the context of data views, over providing the param value directly in the `source`

Reply:
Thanks for the context. I don't have any arguments for it, just curious if it's in our future plan to match with what ES supports but that's out of scope of this PR. I'm happy to resolve this for now.